packages/date-time/src/components/Calendar/CalendarDay/CalendarDay.styles.ts

```diff
@@ -165,9 +158,8 @@ export const styles = (props: ICalendarDayStyleProps): ICalendarDayStyles => {
         width: 24,
         height: 24,
         lineHeight: 24,
+        fontSize: FontSizes.small,
```

Comment:
I moved this fontSize style to the button itself as I noticed it was not reaching the span rendering the date and I assumed this was the intended size of 12px (instead of 11px that is currently). Let me know if I made the correct move here.